sdk/iothub/azure-iot-deviceprovisioning/sdk_packaging.toml

```diff
@@ -0,0 +1,8 @@
+[packaging]
+package_name = "azure-iot-deviceprovisioning"
+package_pprint_name = "IoT Device Provisioning Service"
```

Comment:
Should `package_pprint_name` be "IoT Device Provisioning Client" or "IoT Device Provisioning Service Client"?